src/hooks/useConcentratorRewards.ts

```diff
@@ -31,33 +36,52 @@ export function useConcentratorPendingReward({
     functionName: "pendingReward",
     args: [userAddress ?? "0x"],
   }))
-  return useContractReads({
+  const rewards = useContractReads({
     contracts: contracts,
     enabled: !!userAddress && ybTokenList.length > 0,
     select: (data) => data.map((reward) => BigNumber.from(reward ?? 0)),
   })
+  return rewards
 }
 
 export function useConcentratorClaim({
   targetAsset,
   ybTokenList,
 }: ConcentratorRewardArgs) {
+  const [addToast, replaceToast] = useToastStore(
+    (state) => [state.addToast, state.replaceToast],
+    shallow
```

Comment:
`shallow` is important, glad you caught that. When grabbing multiple items from a store without it, you'll see state updating and thus UI refreshing constantly.